### DISCUSSION TITLE
adding searchpane

### DISCUSSION THREAD

**Discussion by markplus** [nasirkhan/laravel-starter#465] [Ideas]
hello I wanted to know if adding searchpanes to your project is simple and can be done, I've been trying to do it with serviside for some time but I haven't succeeded yet even buying the datatables editor license, can you help me?

**Comment by nasirkhan** [comment]
I have not used this feature of Datatables. So I am not sure how this can be implemented. If you have found the solution, please let me know, I may add that in this project and you may submit a pull request as well.

**Reply by markplus** [reply]
i'm trying but i'm not successful yet, i'm trying to insert export buttons, editor and searchpanes but i'm not able to get it working yet

**Reply by nasirkhan** [reply]
We're you able to solve this issue ?
